test/user-test.js

```diff
@@ -77,17 +77,26 @@ describe('User', function() {
   });
 
   it('should initialize with an id', function() {
-    expect(user1.id).to.eq(1);
+    expect(user1.id).to.equal(1);
   });
 
   it('should initialize with a name', function() {
-    expect(user1.name).to.eq('Saige O\'Kon');
+    expect(user1.name).to.equal('Saige O\'Kon');
   });
 
   it('should initialize with a pantry', function() {
     expect(user1.pantry[0].ingredient).to.eq(11477);
   });
 
+  it('should return 0 if there are no ingredients in the panty', function() {
```

Comment:
Great sad path test! Michael Scarn would be proud.

Reply:
Yes I concur, very good use of sad testing!